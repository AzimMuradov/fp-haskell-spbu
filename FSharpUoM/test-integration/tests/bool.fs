let x = false;
not x;; // Right: "true"

let x = true;
not x;; // Right: "false"

let x = false;
let y = true;
x && y;; // Right: "false"

let x = false;
let y = true;
x || y;; // Right: "true"

let x = false;
let y = true;
x = y;; // Right: "false"

let x = false;
let y = true;
1 <> 2;; // Right: "true"
