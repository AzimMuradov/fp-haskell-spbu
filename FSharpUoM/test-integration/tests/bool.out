val it: bool = True
val it: bool = False
val it: bool = False
val it: bool = True
val it: bool = False
val it: bool = True
